src/partials/_contacts.html

```diff
@@ -1,79 +1,79 @@
 <div class="contacts">
```

Comment:
zamiast diva powinno być section i brakuje jeszcze nagłówka, z tego co pamiętam to w materiałach było napisane, że dla sekcji bez nagłówka trzeba utworzyć niewidoczny nagłówek, mogę podpowiedzieć, że trzeba go będzie stylować identycznie jak etylowaliśmy klasę .is--hidden dla modala w poprzednim HW